packages/asciinema/build.sh

```diff
@@ -2,18 +2,18 @@ TERMUX_PKG_HOMEPAGE=https://asciinema.org/
 TERMUX_PKG_DESCRIPTION="Record and share your terminal sessions, the right way"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2.4.0"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="3.0.0-rc.5"
 TERMUX_PKG_SRCURL=https://github.com/asciinema/asciinema/archive/v${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
For the Rx version to be considered less than 3.0.0 please use a `~`[^1].
```suggestion
TERMUX_PKG_VERSION="3.0.0~rc.5"
TERMUX_PKG_SRCURL=https://github.com/asciinema/asciinema/archive/v${TERMUX_PKG_VERSION/~/-}.tar.gz
```
[^1]: https://wiki.debian.org/Versioning